### PR TITLE
Fix invidious comments

### LINKED ISSUE #235: Can't load comments from an invidious link (TeamNewPipe/NewPipe#2817)
See TeamNewPipe/NewPipe#2817
If you open any invidious link, it won't load comments and will produce and error log

Here is the error log
## Exception
* __User Action:__ requested comments
* __Request:__ https://invidio.us/watch?v=ia1iuXbEaYQ
* __Content Language:__ FR
* __Service:__ YouTube
* __Version:__ 0.18.0
* __OS:__ Linux Android 8.1.0 - 27

<details><summary><b>Crash log</b></summary><p>

```
org.schabi.newpipe.extractor.exceptions.ParsingException: Malformed unacceptable url: https://invidio.us/watch?v=ia1iuXbEaYQ
	at org.schabi.newpipe.extractor.linkhandler.LinkHandlerFactory.fromUrl(LinkHandlerFactory.java:53)
	at org.schabi.newpipe.extractor.linkhandler.ListLinkHandlerFactory.fromUrl(ListLinkHandlerFactory.java:37)
	at org.schabi.newpipe.extractor.linkhandler.ListLinkHandlerFactory.fromUrl(ListLinkHandlerFactory.java:30)
	at org.schabi.newpipe.extractor.StreamingService.getCommentsExtractor(StreamingService.java:259)
	at org.schabi.newpipe.extractor.comments.CommentsInfo.getInfo(CommentsInfo.java:25)
	at org.schabi.newpipe.util.ExtractorHelper.lambda$getCommentsInfo$6(ExtractorHelper.java:139)
	at org.schabi.newpipe.util.-$$Lambda$ExtractorHelper$4SXML3L5j2DhMfS2ZK06RZb2iyE.call(Unknown Source:4)
	at io.reactivex.internal.operators.single.SingleFromCallable.subscribeActual(SingleFromCallable.java:44)
	at io.reactivex.Single.subscribe(Single.java:3438)
	at io.reactivex.internal.operators.single.SingleDoOnSuccess.subscribeActual(SingleDoOnSuccess.java:35)
	at io.reactivex.Single.subscribe(Single.java:3438)
	at io.reactivex.internal.operators.maybe.MaybeFromSingle.subscribeActual(MaybeFromSingle.java:41)
	at io.reactivex.Maybe.subscribe(Maybe.java:4154)
	at io.reactivex.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.drain(MaybeConcatArray.java:153)
	at io.reactivex.internal.operators.maybe.MaybeConcatArray$ConcatMaybeObserver.request(MaybeConcatArray.java:78)
	at io.reactivex.internal.operators.flowable.FlowableElementAtMaybe$ElementAtSubscriber.onSubscribe(FlowableElementAtMaybe.java:66)
	at io.reactivex.internal.operators.maybe.MaybeConcatArray.subscribeActual(MaybeConcatArray.java:42)
	at io.reactivex.Flowable.subscribe(Flowable.java:14479)
	at io.reactivex.internal.operators.flowable.FlowableElementAtMaybe.subscribeActual(FlowableElementAtMaybe.java:36)
	at io.reactivex.Maybe.subscribe(Maybe.java:4154)
	at io.reactivex.internal.operators.maybe.MaybeToSingle.subscribeActual(MaybeToSingle.java:46)
	at io.reactivex.Single.subscribe(Single.java:3438)
	at io.reactivex.internal.operators.single.SingleSubscribeOn$SubscribeOnObserver.run(SingleSubscribeOn.java:89)
	at io.reactivex.Scheduler$DisposeTask.run(Scheduler.java:578)
	at io.reactivex.internal.schedulers.ScheduledRunnable.run(ScheduledRunnable.java:66)
	at io.reactivex.internal.schedulers.ScheduledRunnable.call(ScheduledRunnable.java:57)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:301)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1162)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:636)
	at java.lang.Thread.run(Thread.java:764)

```
</p></details>
<hr>
How to reproduce:
pick any invidious link and open it with newpipe (either you pick an invidious link, or you pick a youtube link and make it invidious link by changing youtube.com by invidio.us)

### PR DESCRIPTION
**Pull request by B0pol**
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [x] I did test the API against [NewPipe](https://github.com/TeamNewPipe/NewPipe).
~~- [] I agree to ASAP create a PULL request for [NewPipe](https://github.com/TeamNewPipe/NewPipe) for making in compatible when I changed the api.~~ doesn't require any changes (except updating the version ofc)

Fixes #235, fixes TeamNewPipe/NewPipe#2817

Edit: apk if you wanna test: [invidious_comments.zip](https://github.com/TeamNewPipe/NewPipeExtractor/files/4076851/invidious_comments.zip)

### CODE REVIEW

**Comment by TobiGr** [comment]
Nice a bug fix which removes code :D

File: extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubeCommentsExtractorTest.java [hunk 1]
```diff
@@ -41,13 +55,20 @@ public void testGetComments() throws IOException, ExtractionException {
...
     public void testGetCommentsFromCommentsInfo() throws IOException, ExtractionException {
+        assertTrue(getCommentsFromCommentsInfoHelper(urlYT));
+        assertTrue(getCommentsFromCommentsInfoHelper(urlInvidious));
+        assertTrue(getCommentsFromCommentsInfoHelper(urlInvidioush));
+    }
+
+    private boolean getCommentsFromCommentsInfoHelper(String url) throws IOException, ExtractionException {
         boolean result = false;
-        CommentsInfo commentsInfo = CommentsInfo.getInfo("https://www.youtube.com/watch?v=D00Au7k3i6o");
+        CommentsInfo commentsInfo = CommentsInfo.getInfo(url);
```
Comment (TobiGr) [inline extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubeCommentsExtractorTest.java]:
``` Java
assertTrue("what the fuck am i doing with my life".equals(commentsInfo.getName()));
```
Can you convert this to `assertEquals()` please?

Comment (B0pol) [reply extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubeCommentsExtractorTest.java]: done

Comment (TobiGr) [reply extractor/src/test/java/org/schabi/newpipe/extractor/services/youtube/YoutubeCommentsExtractorTest.java]: thanks

**Pull request merged**